src/avram/polymorphic.cr

```diff
@@ -0,0 +1,144 @@
+# Helpers for defining polymorphic associations
+module Avram::Polymorphic
+  # Sets up helper methods for validating and loading polymorphic associatiions
+  #
+  # This will generate methods for validiating, preloading, and lazy loading
+  # polymorphic associations.
+  #
+  # Let's start with this example:
```

Comment:
I think I'd put this below the `Important note:`

Reply:
Good idea